monkey/infection_monkey/post_breach/signed_script_proxy/windows/signed_script_proxy.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+
+ORIGINAL_COMSPEC = r'C:\Windows\System32\cmd.exe'
+
+
+def get_windows_commands_to_proxy_execution_using_signed_script():
+    global ORIGINAL_COMSPEC
```

Comment:
Not loving the global here. Let's use the PBA instance state to manage this instead?